tests/distributions/test_von_mises.py

```diff
@@ -9,10 +9,53 @@
 from torch import optim
 
 from pyro.distributions import VonMises, VonMises3D
-from pyro.distributions.von_mises import _log_modified_bessel_fn
 from tests.common import skipif_param
 
 
+def _eval_poly(y, coef):
```

Comment:
This testing code was moved from `von_mises.py` which was removed.

Reply:
Should we move all of these tests to pytorch too? cc. @ahmadsalim

Reply:
Yes, I can do that 😄

Reply:
@ahmadsalim I think it's fine for those tests to live in Pyro for now. I'm pretty unhappy with PyTorch's test_distributions.py file: it is huge, monolithic, and uses poor testing practices. I'd like to split it up and start using PyTest. After we do that it should be easy to simply copy these tests over rather than rewrite them to avoid pytest (and later have to rewrite them to use pytest).

Reply:
@neerajprad maybe we can put some time into splitting up PyTorch's test_distributions.py and starting to use pytest there https://github.com/pytorch/pytorch/issues/11578 , and afterwards move @ahmadsalim 's tests?

Reply:
That sounds good. It will be worth reorganizing our monolithic `test_distributions.py` using pytest first before adding more to the file. Most pytorch devs already use pytest as a test runner, but for some reason there has been reluctance to add it as a test dependency and use all  the testing utilities that it offers.